From 89a8edde6c00a67281cf652f7bf9ba21350b9f20 Mon Sep 17 00:00:00 2001
From: Heiko Schocher <hs@nabladev.com>
Date: Mon, 3 Oct 2022 07:37:51 +0200
Subject: [PATCH 18/23] add our own version

add file tbot/_version.py to have our own version string

Base is: v0.10.4

Signed-off-by: Heiko Schocher <hs@nabladev.com>
---
 tbot/_version.py | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 tbot/_version.py

diff --git a/tbot/_version.py b/tbot/_version.py
new file mode 100644
index 0000000..1b48fa5
--- /dev/null
+++ b/tbot/_version.py
@@ -0,0 +1 @@
+version = "0.10.10-patched"
-- 
2.31.1

